_posts/2023-11-20-sct-course.md

```diff
@@ -9,11 +9,13 @@ location: Online
 
 Course on the Spinal Cord Toolbox (SCT) software.
 
+The general aim of this 2-day course is to learn how to analyse spinal cord quantitative MRI data. There will first be an introduction presenting the challenges/solutions of spinal cord MRI, followed by a general overview of the Spinal Cord Toolbox (SCT) and its applications in various diseases. Then, the course will dive into the various processes of spinal cord MRI analysis, from segmentation, to the registration to a template, to the extraction of quantitative metrics (diffusion MRI, magnetization transfer). We will also touch base on various "niche" applications, such as the quantification of compression in spinal cord injured patients, fMRI analysis, etc. We will also discuss "best practice" for running reproducible analyses using batch scripts. 
+
```

Comment:
Maybe we can also include how to run a [multi-subject analysis](https://spinalcordtoolbox.com/user_section/tutorials/analysis-pipelines-with-sct/sct_run_batch-usage.html), how to do [quality control](https://spinalcordtoolbox.com/user_section/tutorials/analysis-pipelines-with-sct/inspecting-the-results.html#quality-control-report), and how to perform [manual corrections](https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4155).

Reply:
thanks! done in [00d57d5](https://github.com/spinalcordmri/spinalcordmri.github.io/pull/82/commits/00d57d5e5ab5446d7ddd00cd112169a14132631c)